Tests/Apollo/Network/HttpNetworkTransportTests.swift

```diff
@@ -0,0 +1,342 @@
+import XCTest
+@testable import Apollo
+import ApolloTestSupport
+import StarWarsAPI
+
+class HttpNetworkTransportTests: XCTestCase {
```

Comment:
I think this is more "Automatic Persisted Queries Tests"  than something generic about the network transport - do you mind renaming?

Reply:
updated